fury/ui/elements.py

```diff
@@ -3272,3 +3272,696 @@ def directory_click_callback(self, i_ren, _obj, listboxitem):
                 self.set_slot_colors()
         i_ren.force_render()
         i_ren.event.abort()
+
+
+class Tree2D(UI):
```

Comment:
you should add `Tree2D` in the `__all__` at the top of the file

Reply:
Done.

Reply:
Done.